### PR TITLE
buttons in get started message added

### PR DESCRIPTION
**Pull request by Manveer695**
Fixes issue #21 i.e. introduce buttons to the SUSI Tweet bot.

Screenshots for the change: 
getstatedtwitter

### CODE REVIEW

**Comment by Manveer695** [comment]
@AliAyub007 @chashmeetsingh @dynamitechetan @betterclever Please review! Thanks!

**Review by betterclever — APPROVED** [review]

**Review by chashmeetsingh — APPROVED** [review]

**Review by mariobehling — COMMENTED** [review]
Where does this message come from? All messages should be coming from the server and should be constituted as a skill.

The welcome message is erroneous and FOSSASIA is always spelled in capitals. (A pity you don't know that after several months of GSoC).

**Review by mariobehling — CHANGES_REQUESTED** [review]
Please ensure that messages come from the server. Also ensure correct spelling and grammar.

**Comment by Manveer695** [comment]
@mariobehling Made a [skill](https://github.com/fossasia/susi_skill_data/pull/257) in susi_skill_data repo, to get the answers from server for every query. Thank you, sir.

File: index.js [hunk 1]
```diff
@@ -109,58 +110,88 @@ function TwitterBot() {
...
-							message += 'Universities are shown below-:\n\n';
-						for(var i=0;i<((data.answers[0].metadata.count)>50?50:data.answers[0].metadata.count);i++){
-							for(var cN in colNames){
-								message += (colNames[cN]+' : ');
-								message += data.answers[0].data[i][cN]+', ';
+			});		
+			sendMessage(senderName, text);
+		}
+		else if(directMsg.direct_message.text === "View Repository"){
+			sendMessage(senderName, "Visit https://www.github.com/fossasia/susi_server");
```
Comment (mariobehling) [inline index.js]: Why is this hardcoded here?

**Comment by Manveer695** [comment]
@mariobehling Removed the hardcode, sir. Now, clicking on the View Repository button will redirect us directly to the SUSI server repository. Thank you.

**Review by mariobehling — APPROVED** [review]

**Comment by mariobehling** [comment]
Resolve conflicts!

**Comment by Manveer695** [comment]
@mariobehling Thanks a lot for the approved review. Resolved the conflicts, sir. Thank you.

**Comment by Manveer695** [comment]
@mariobehling Please merge it, sir. I need it for further work. Thank you.

**Comment by Manveer695** [comment]
@mariobehling Due to new PR's getting merged, conflicts arise in this PR. I have resolved the conflicts. Please merge it, I need it for further work. Thank you.

**Pull request merged**